Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -21,7 +21,7 @@ extension FileManager: LintableFileManager {
             .standardizingPath
 
         // if path is a file, it won't be returned in `enumerator(atPath:)`
-        if absolutePath.bridge().isSwiftFile() && absolutePath.isFile {
+        if absolutePath.isFile {
```

Comment:
when specifying `included` and `excluded` configuration values, which IMO doesn't really make sense to specify non-Swift files in there.